guides/source/active_record_migrations.md

```diff
@@ -1083,32 +1310,65 @@ This will generate the following output:
+migration, stored in the `version` column. This allows Rails to determine which
+migrations have already been applied to the database.
+
+For example, if you have a migration file named 20240428000000_create_users.rb,
+Rails will extract the version number (20240428000000) from the filename and
+insert it into the schema_migrations table after the migration has been
+successfully executed.
+
+You can view the contents of the schema_migrations table directly in your
+database management tool or by using Rails console:
+
+```irb
+rails dbconsole
+```
 
-[`say`]: https://api.rubyonrails.org/classes/ActiveRecord/Migration.html#method-i-say
-[`say_with_time`]: https://api.rubyonrails.org/classes/ActiveRecord/Migration.html#method-i-say_with_time
-[`suppress_messages`]: https://api.rubyonrails.org/classes/ActiveRecord/Migration.html#method-i-suppress_messages
+Then, within the database console, you can query the schema_migrations table:
+
+```sql
+SELECT * FROM schema_migrations;
+```
+
+This will show you a list of all migration version numbers that have been
+applied to the database. Rails uses this information to determine which
+migrations need to be run when you run rails db:migrate or rails db:migrate:up
+commands.
 
 Changing Existing Migrations
 ----------------------------
 
 Occasionally you will make a mistake when writing a migration. If you have
 already run the migration, then you cannot just edit the migration and run the
 migration again: Rails thinks it has already run the migration and so will do
-nothing when you run `bin/rails db:migrate`. You must rollback the migration (for
-example with `bin/rails db:rollback`), edit your migration, and then run
+nothing when you run `bin/rails db:migrate`. You must rollback the migration
+(for example with `bin/rails db:rollback`), edit your migration, and then run
 `bin/rails db:migrate` to run the corrected version.
 
-In general, editing existing migrations is not a good idea. You will be
-creating extra work for yourself and your co-workers and cause major headaches
-if the existing version of the migration has already been run on production
-machines.
+In general, editing existing migrations is not a good idea. You will be creating
+extra work for yourself and your co-workers and cause major headaches if the
+existing version of the migration has already been run on production machines.
 
-Instead, you should write a new migration that performs the changes
-you require. Editing a freshly generated migration that has not yet been
-committed to source control (or, more generally, which has not been propagated
-beyond your development machine) is relatively harmless.
+Instead, you should write a new migration that performs the changes you require.
+Editing a freshly generated migration that has not yet been committed to source
+control (or, more generally, which has not been propagated beyond your
+development machine) is relatively harmless.
```

Comment:
Oh looks like there is something about my above comment. So maybe moving that up? Otherwise it seems like we're contradicting ourselves.